ols/app/endpoints/ols.py

```diff
@@ -81,6 +114,8 @@ def conversation_request(llm_request: LLMRequest) -> LLMRequest:
             )
 
             # Summarize documentation
+            # TODO: consider moving DocsSummarizer to query_helpers and
+            # use its interface
```

Comment:
I didn't touch the `DocsSummarized` yet. It uses values from the config. See the thread in [217](https://github.com/openshift/lightspeed-service/pull/217).